lib/pages/demo.dart

```diff
@@ -712,7 +715,18 @@ class DemoContent extends StatelessWidget {
           top: Radius.circular(10.0),
           bottom: Radius.circular(2.0),
         ),
-        child: DemoWrapper(child: Builder(builder: buildRoute)),
+        child: Theme(
+          data: MaterialDemoThemeData.themeData.copyWith(
+            platform: GalleryOptions.of(context).platform,
+          ),
+          child: CupertinoTheme(
+            data: const CupertinoThemeData()
+                .copyWith(brightness: Brightness.light),
+            child: ApplyTextOptions(
```

Comment:
In the old code for DemoWrapper above, the CupertinoTheme was the child of ApplyTextOptions, this switches them around. Is there any reason for this? or does it not matter?